src/vos/tests/evt_ctl.sh

```diff
@@ -167,6 +158,11 @@ echo "$cmd"
 
 $cmd -t
 
-result=$?
-echo Test returned $result
-exit $result
+if [[ -z ${CMOCKA_MESSAGE_OUTPUT+x} || -z ${CMOCKA_XML_FILE+x} ]]; then
+	result=$?
```

Comment:
Hrm.  I made this comment in the previous review but it seems to have gotten dropped.  In any case, `${PIPESTATUS[0]}` is preferred to `$?` because the command/pipeline you are testing the return status of doesn't change if a `| $command` is added to it in the future.

```suggestion
	result="${PIPESTATUS[0]}"
```

Reply:
Updated